error/s2n_errno.c

```diff
@@ -287,10 +288,14 @@ static const char *no_such_error = "Internal s2n error";
     ERR_ENTRY(S2N_ERR_INTERNAL_LIBCRYPTO_ERROR, "An internal error has occurred in the libcrypto API") \
     ERR_ENTRY(S2N_ERR_NO_RENEGOTIATION, "Only secure, server-initiated renegotiation is supported") \
     ERR_ENTRY(S2N_ERR_APP_DATA_BLOCKED, "Blocked on application data during handshake") \
-/* clang-format on */
+    /* clang-format on */
```

Comment:
Is this clang-format off/on still needed?

Reply:
Yeah. No -- col limit 0 to the rescue!

Reply:
Well yes. We align newline backslashes and one of these lines is very long.